lib/i18n/index.ts

```diff
@@ -9,8 +9,9 @@ declare module 'i18next' {
 }
 
 // Skeleton instance will enable building i18n ready components
-function configure() {
+export function configure() {
   return i18next.use(initReactI18next).init({
+    compatibilityJSON: 'v3',
```

Comment:
Without this, I got [errors about requiring a plural polyfill](https://github.com/i18next/i18next/issues/1671).  Let's hold off on that (polyfills) for now.